managed/services/dump/metrics.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2023 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package dump
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	prom "github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm/managed/models"
+)
+
+const (
+	requestTimeout              = 3 * time.Second
+	dumpExists          float64 = 1
+	prometheusNamespace         = "pmm_managed"
+	prometheusSubsystem         = "dumps"
+)
+
+type MetricsCollector struct {
+	db *reform.DB
+	l  *logrus.Entry
+
+	mDumpsDesc *prom.Desc
+}
+
+func NewMetricsCollector(db *reform.DB) *MetricsCollector {
+	return &MetricsCollector{
+		db: db,
+		l:  logrus.WithField("component", "dumps/metrics"),
+		mDumpsDesc: prom.NewDesc(
+			prom.BuildFQName(prometheusNamespace, prometheusSubsystem, "dumps"),
+			"Dumps",
+			[]string{
+				"dump_id", "status",
```

Comment:
do we really need dump_id in metrics? shouldn't we show more related metrics like the ones we want to provide in telemetry?

Reply:
In fact we don't need this metrics at all. I will remove this file completely for now.

Reply:
P.S. this metric were copied from backups, there we use it for alerting

Reply:
Removed metrics.go